oaknational/Oak-Web-Application
# chore(fixes-spacing): fix spacing on mobile view and heading sizes

## Description

- Changes the margin on desktop on mobile to match Figma
- Changes the title heading also to match Figma

## Issue(s)

Fixes #1568 

## How to test

1. Go to https://deploy-preview-1571--oak-web-application.netlify.app/beta/teachers/programmes/maths-secondary-ks4-core/units/standard-form-writing-and-converting-f42a/lessons/convert-large-standard-form-numbers-to-ordinary-form-6xk36e

You should see spacing fixed for mobiles and header changes size also

## Screenshots

How it used to look (delete if n/a):
![image](https://user-images.githubusercontent.com/91190841/235906458-fca7d54c-91cd-4fee-83bc-bb3ea9508be7.png)


How it should now look:

# Mobile 
<img width="452" alt="Screenshot 2023-05-03 at 12 47 46" src="https://user-images.githubusercontent.com/91190841/235907404-90845dac-ecdc-4b9d-9484-5f28a1c5c304.png">

# Desktop
<img width="787" alt="Screenshot 2023-05-03 at 12 23 28" src="https://user-images.githubusercontent.com/91190841/235907505-f115e3c4-d0b8-4ce1-96ff-cf975aad7474.png">


## Checklist
